package.json

```diff
@@ -22,5 +22,8 @@
     "not dead",
     "not ie <= 11",
     "not op_mini all"
-  ]
+  ],
+  "devDependencies": {
+    "webpack-dev-server": "^3.1.9"
```

Comment:
I think this shouldn't be added as CRA already ships all needed devDependancies with react-scripts